src/students/views.py

```diff
@@ -36,8 +38,10 @@ def groups(request):
     fn = request.GET.get('name')
     if fn:
         queryset = queryset.filter(name__contains=fn)
+        queryset = queryset.filter(name__contains=fn)
     for group in queryset:
-        response += group.get_info() + '<br>' + '<br>'
+        response += group.get_info() + f'<a href="{reverse("groups-edit-for-link")}{group.id}">' \
```

Comment:
`f'<a href="{reverse("groups-edit-for-link")}{group.id}">' -> f'<a href="{reverse("groups-edit", args=[group.id])}">'`